scripts/xapi-tracing-log-trim.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+export MAX_MB=100
+
+# find the oldest trace files beyond MAX_MB size and delete
+find /var/log/dt -type f -printf "%T@ %s %p\n" | sort -nr | awk 'BEGIN{size=0}{size+=$2; if (size>1024*1024*MAX_MB) {$1=""; $2=""; print $0}}' | xargs rm
+
+# delete any remaining files that are older than 30 days
```

Comment:
would be good to parametrize the script (with a default of 30 days), so we can test in XenRT without having to wait for 30days...

Reply:
After the latest changes, I can test the script successfully in dom0 using eg.
```
MAX_MB=10 /opt/xensource/libexec/xapi-tracing-log-trim.sh  #remove the oldest files beyond 10MB total in dir
MAX_DAYS=0 /opt/xensource/libexec/xapi-tracing-log-trim.sh #remove all the files immediately
```